plone/app/upgrade/v51/profiles/to_514/registry.xml

```diff
@@ -115,13 +115,13 @@
       prefix="plone.bundles/plone"
       interface="Products.CMFPlone.interfaces.IBundleRegistry"
       purge="False">
-    <value key="last_compilation">2018-09-06 00:00:00</value>
+    <value key="last_compilation">2018-09-14 00:00:00</value>
```

Comment:
I have just compiled the plone-logged-in bundle

Reply:
@erral ok:) I will compile both as a result of https://github.com/plone/mockup/pull/863

Reply:
(the datatables css -> inline fixes)

Reply:
Ok then.